tokamak/1-no-currents/BOUT.inp

```diff
@@ -0,0 +1,309 @@
+
+NOUT = 100
+TIMESTEP = 1000
+
+TwistShift = false  # use twist-shift condition?
+ballooning = true
+shiftinitial = false
+
+MZ = 1     # number of points in z direction (2^n + 1)
+
+# NXPE = 10
+
+ZPERIOD=1
+
+grid = tokamak-example.nc
+
+[restart]
+init_missing = true
+
+[mesh]
+paralleltransform = shifted
+
+symmetricGlobalX = true
+
+shiftangle_from_zshift=true
+
+extrapolate_y = false
+
+##################################################
+# derivative methods
+
+[ddx]
+
+first = C2
+second = C2
+upwind = W3
+
+[ddy]
+
+first = C2
+second = C2
+upwind = W3
+
+[ddz]
+
+first = C2
+second = C2
+upwind = W3
+
+###################################################
+# Time-integration solver
+
+[solver]
+mxstep = 1000000
+
+# Note: If evolving neutrals, need preconditioning
+type = cvode
+use_precon = true
+
+maxl = 10
+
+ATOL = 1.0e-10  # absolute tolerance
+RTOL = 1.0e-5   # relative tolerance
+mxstep = 1000000  # Maximum internal steps per output
+
+[laplace]  # This is used for Delp2 operator
+all_terms = true
+nonuniform=true
+
+##################################################
+# Electrostatic potential solver
+
+[phiSolver]
+inner_boundary_flags = 0
+outer_boundary_flags = 16  # INVERT_SET  (2.8 * Te)
+
+all_terms = true
+nonuniform=true   # NOTE: Necessary to avoid numerical instability
+
+[laplacexy]  # 2D solver in X-Y
+pctype = hypre    # Preconditioner
+
+atol = 1e-12
+rtol = 1e-8
+
+core_bndry_dirichlet = false
```

Comment:
In case at some point the input file is modified to turn currents on - if we expect to use the evolving phi boundary conditions, should this be `true`?

Reply:
Currently set to zero-gradient in the core, then evolving just outer boundary. Definitely should note this somewhere...